lib/gui/pages/settings/templates/settings.tpl.html

```diff
@@ -48,7 +48,7 @@ <h3 class="subtitle">Advanced</h3>
         ng-model="settings.currentData.unsafeMode"
         ng-change="settings.enableDangerousSetting('unsafeMode', {
           description: 'Are you sure you want to turn this on? You will be able to overwrite your system drives if you\'re not careful.',
-          confirmationLabel: 'Enable unsafe mode'
+          confirmationLabel: 'Enable unsafe mode', rejectionLabel: 'Cancel'
```

Comment:
Was this change (adding the Cancel button back to the "enable unsafe" popup) **supposed** to be included in this PR too? :-/